.github/workflows/linux.yml

```diff
@@ -142,7 +142,7 @@ jobs:
           # For some reason, building with 'RelWithDebInfo' (default) doesn't work 
           # specifically on PRs, so we'll just build Releases for everything for now
           #if [ '${{ github.event_name }}' != 'pull_request' ]; then
-            RELEASE_TYPE="-R"

```

Comment:
I don't think this is doing what you think :-)

-R is a flag that the build script uses, so it should only ever be set when releases are made. In this case what should be done is the `if` line above should be uncommented, and the `fi` below. This should stay as -R for builds on master IMO, but I think we need to decide what releases happen where and stuff anyway. As is, this with the comments removed will build release builds for `master` and when we do actual releases via tags. Everything else (which should just be PRs) will be relwithdebinfo.

Reply:
Actually, I think we should always do RelWithDebInfo,unless maybe we ship debug symbols separately

Reply:
Yeah, we should probably make that decision with the group on Discord. I don't have a very strong opinion either way really. If we decide to always do ship with debug then obviously this section can be removed.